### PR TITLE
P55. exercise 55

### PR DESCRIPTION
**Pull request by audtjddld**
i pull request again

remove unnecessary parameter count

### CODE REVIEW

**Comment by loustler** [comment]
@audtjddld Plz Add test code

**Comment by loustler** [comment]
This PR reference by #15

**Comment by audtjddld** [comment]
i do not add test code to commit. so that i retry pull request. -_-;

File: src/main/scala/com/chobostudy/audtjddld/datastructure/SolutionP55.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.chobostudy.loustler.datastructure
```
Comment (loustler) [inline src/main/scala/com/chobostudy/audtjddld/datastructure/SolutionP55.scala]: I think this package path is incorrect, right?

File: src/test/scala/com/chobostudy/audtjddld/datastructure/SolutionP55Test.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.chobostudy.loustler.datastructure
```
Comment (loustler) [inline src/test/scala/com/chobostudy/audtjddld/datastructure/SolutionP55Test.scala]: It's too 

The package path is incorrect.

**Review by loustler — CHANGES_REQUESTED** [review]
A some errors occurs from your test code.

Please fix 

The incorrect package path which reviewed by me in this review to occur error.

**Comment by audtjddld** [comment]
I modified the package path and converted it to a gradle project.

**Comment by audtjddld** [comment]
In the Eclipse IDE, it was a previously unrecognized Scala project for the git repository.
So I set up the scala project in the `.project` file and applied the gradle to add the missing `org.scalatest` library.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		java {
+			srcDirs = ["src/main/scala", "src/test/scala"]
+		}    
+    }
+}
+
+dependencies {
+    compile group: 'org.scala-lang', name: 'scala-library', version: '2.12.2'
+    testCompile 'org.scalatest:scalatest_2.12:3.0.1'
+  	testRuntime 'org.pegdown:pegdown:1.4.2'
```
Comment (loustler) [inline build.gradle]: reindent plz

**Review by loustler — CHANGES_REQUESTED** [review]
Did you have any reason why use gradle ?

File: build.gradle (hunk 3 above)
Comment (audtjddld) [reply build.gradle]: done

Comment (audtjddld) [reply build.gradle]: missing org.scalatest library.

**Review by loustler — APPROVED** [review]
I'll merge it.

Review #9 plz

**Pull request merged**